packages/flutter/test/widgets/autocomplete_test.dart

```diff
@@ -637,7 +637,7 @@ void main() {
     expect(optionsOffsetOpen.dy, fieldOffset.dy + fieldSize.height);
   });
 
-  testWidgets('can prevent options from showing by returning an empty iterable', (WidgetTester tester) async {
+  testWidgets('does not prevent options from showing when returning an empty iterable', (WidgetTester tester) async {
```

Comment:
This is a behavior change that might not always be desired. What if someone *wants* to prevent the options from showing when `optionsBuilder` returns an empty list? It seems reasonable that they should be able to do that, and it seems risky to change the behavior on people that do.

Maybe the best bet does end up being to make this possibly breaking change, but we should be sure about it, and sure that people can still prevent the options from showing when they want to.

Reply:
Rather than changing this test, maybe there should be 2 tests. One showing how to show the options when the list is empty and one showing how to hide them.